prune-releases.sh

```diff
@@ -9,6 +9,7 @@ function usage()
 {
     echo "$0"
     echo "    -h --help"
+    echo '    --max-releases=10'
```

Comment:
Can we add text to this? Is it `max releases to keep` or `max releases to delete` ?

Reply:
good call!